src/quantum/subbenchmarks.jsx

```diff
@@ -38,6 +41,26 @@ const Graph = ({ data, name, url }) => (
         />
       </a>
     </div>
+    <ChartJsWrapper
+      title={name}
+      type='line'
+      height={80}
+      options={generateOptions()}
```

Comment:
Use `getPerferherderData` and you can pass the `options` it generates:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/containers/PerfherderGraphContainer/index.jsx#L30

Reply:
from what I know `getPerferherderData` can not get subtest data.  Is there some other place that the styting information can come from?